packages/mesa.rb

```diff
@@ -17,10 +17,10 @@ class Mesa < Package
      x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mesa/21.3.4_x86_64/mesa-21.3.4-chromeos-x86_64.tpxz'
   })
   binary_sha256({
-    aarch64: 'ab9a19dd501715367d45e843b64b2aec16741538caa033f61133d9e78eca68df',
-     armv7l: 'ab9a19dd501715367d45e843b64b2aec16741538caa033f61133d9e78eca68df',
-       i686: '9d70ca4db18e06c3d63f867801171e3fc960b841a9e62218ce3ff91b3036bd63',
-     x86_64: 'bf3dff20994cdd88a1a242351c21bea4ce111410171ea5d39728fd6406ea40bf'
+    aarch64: '5e018f58ac7261a46c8e6e133f26efa3e20d3daa7321584c70b1fdc4be6c9c71',
+     armv7l: '5e018f58ac7261a46c8e6e133f26efa3e20d3daa7321584c70b1fdc4be6c9c71',
+       i686: 'f67e9f749982cf67ca6942d0ec25d9b623ecfb687434e89d178edc2f05234b0c',
+     x86_64: '05cf28be056481799ea5458fdb050f33c5410ccf3e8ffc63b0dee02cdafef450'
   })
```

Comment:
Should we add tests with `def self.check`?

Reply:
yes probably

Reply:
New binaries needed because of #6700?

Reply:
Or just use the binaries in https://github.com/skycocker/chromebrew/pull/6700 since it has the libosmesa wine needs anyways...